packages/core/src/button/register.ts

```diff
@@ -10,6 +10,11 @@ import { registerElementSafely } from '@cds/core/internal';
 import { CdsButton } from './button.element.js';
 import { CdsIconButton } from './icon-button.element.js';
 import { CdsInlineButton } from './inline-button.element.js';
+import { ClarityIcons } from '@cds/core/icon/icon.service.js';
+import { errorStandardIcon } from '@cds/core/icon/shapes/error-standard.js';
+import { checkIcon } from '@cds/core/icon/shapes/check.js';
+
+ClarityIcons.addIcons(errorStandardIcon, checkIcon);
```

Comment:
Moving the side effects to the register to follow the rest of the component register files